### PR TITLE
Support Wagtail 5.0

### PR DESCRIPTION
**Pull request by victoriachan**
This MR also includes the previous MR (#4) for Wagtail 4.2 support, by @nickmoreton

### CODE REVIEW

**Comment by victoriachan** [comment]
@alxbridge Are you happy to merge this in?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+- Add support for wagtail = 5.0
+- Add support for Wagtail 4 and drop django versions before 3.2 and allow django 4.1
+- Add support for wagtail >= 4.1
+- Drop support for wagtail < 4.1
```
Comment (alxbridge) [inline CHANGELOG.md]: **chore:** Update this section to follow keepachangelog format, e.g.

```suggestion
### Added

- Support for Wagtail versions >= 4.1 (including 5.0)
- Support for Django versions 4.1 and 4.2

### Removed

- Support for Django versions < 3.2
- Support for Wagtail versions < 4.1
```

File: setup.py [hunk 2]
```diff
@@ -31,7 +34,6 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        "wagtail>=2.15",
-        "django>=3.0,<4.0"
+        "wagtail>=4.1",
```
Comment (alxbridge) [inline setup.py]: **issue:** This definition allows installation with versions of Wagtail which we don't yet know are compatible - add an upper limit.

```suggestion
        "wagtail>=4.1,<6.0",
```

File: wagtail_purge/wagtail_hooks.py [hunk 3]
```diff
@@ -22,4 +19,4 @@ def register_purge_menu_item():
         reverse("purge"),
         classnames="icon icon-collapse-down",
         order=1000,
-    )
\ No newline at end of file
+    )
```
Comment (alxbridge) [inline wagtail_purge/wagtail_hooks.py]: **nitpick:** Restore the newline to the end of this file, to follow coding standards consistently

**Pull request merged**